src/function/gds/k_core_decomposition.cpp

```diff
@@ -16,53 +16,37 @@ using namespace kuzu::graph;
 namespace kuzu {
 namespace function {
 
-class CoreValues {
-public:
-    CoreValues(const table_id_map_t<offset_t>& maxOffsetMap, MemoryManager* mm) {
-        init(maxOffsetMap, mm);
-    }
-
-    void pinTable(table_id_t tableID) { coreValues = coreValuesMap.getData(tableID); }
-
-    bool isValid(offset_t offset) {
-        return coreValues[offset].load(std::memory_order_relaxed) != INVALID_DEGREE;
-    }
-    degree_t getValue(offset_t offset) {
-        return coreValues[offset].load(std::memory_order_relaxed);
-    }
-    void setCoreValue(offset_t offset, degree_t value) {
-        coreValues[offset].store(value, std::memory_order_relaxed);
-    }
+struct KcoreComputationState {
```

Comment:
Revert the naming to CoreValues. Otherwise I find KcoreComputationState.isValid to be very confusing.